src/router/index.js

```diff
@@ -1,73 +1,92 @@
 import { createRouter, createWebHistory } from "vue-router";
+// import { useStore } from "vuex";
 const Home = () => import("../views/Home.vue");
 const Resources = () => import("../views/Resources.vue");
 const Products = () => import("../views/Products.vue");
 const Users = () => import("../views/Users.vue");
 const ResourceDetails = () => import("../views/ResourceDetails.vue");
 const UserDetails = () => import("../views/UserDetails.vue");
 const UserResourceDetails = () => import("../views/UserResourceDetails.vue");
-
+const Login = () => import("../views/Login.vue");
+const NotFound = () => import("../views/NotFound.vue");
+// const store = useStore();
+import store from "@/store/store";
 const routes = [
-  { path: "/", redirect: "/home" },
+  { path: "/", redirect: "/login" },
+  {
+    path: "/login",
+    name: "Login",
+    component: Login,
+    meta: { title: "Login page", requiresunAuth: true },
+  },
   {
     path: "/home",
     name: "Home",
     component: Home,
-    meta: { title: "Home page" },
+    meta: { title: "Home page", requiresAuth: true },
   },
   {
     path: "/users",
     name: "Users",
     component: Users,
-    meta: { title: "Users page" },
+    meta: { title: "Users page", requiresAuth: true },
   },
   {
     path: "/users/:id",
     props: true,
     name: "Users Details",
     component: UserDetails,
-    meta: { title: "Users Details" },
+    meta: { title: "Users Details", requiresAuth: true },
   },
   {
     path: "/resources/availability/add/:id",
     props: true,
     name: "Add-Quantity",
     component: UserResourceDetails,
-    meta: { title: "Add Quantity" },
+    meta: { title: "Add Quantity", requiresAuth: true },
   },
   {
     path: "/resources/availability/remove/:id",
     props: true,
     name: "Remove-Quantity",
     component: UserResourceDetails,
-    meta: { title: "Remove Quantity" },
+    meta: { title: "Remove Quantity", requiresAuth: true },
   },
   {
     path: "/resources",
     name: "Resources",
     component: Resources,
-    meta: { title: "Resources page" },
+    meta: { title: "Resources page", requiresAuth: true },
   },
   {
     path: "/resources/add",
     name: "Add Resources",
     component: ResourceDetails,
-    meta: { title: "Add resource" },
+    meta: { title: "Add resource", requiresAuth: true },
   },
   {
     path: "/resources/edit/:id",
     name: "Edit-Resource",
     props: true,
     component: ResourceDetails,
-    meta: { title: "Edit resource" },
+    meta: { title: "Edit resource", requiresAuth: true },
   },
 
   {
     path: "/products",
     name: "Products",
     component: Products,
-    meta: { title: "Products page" },
+    meta: { title: "Products page", requiresAuth: true },
   },
+
+  {
+    path: "/logout",
+    name: "Logout",
+    component: Products,
+    meta: { title: "Logout page", requiresAuth: true },
```

Comment:
Do we need this? What are we showing on this page? It's a bit strange that we navigate here on a logout and requiresAuth is true. I may be misunderstanding this

Reply:
fixed now but i faced issue with router.push('/login') after clicking logout on SM devices, turns out it goes to 404 for some reason, so i removed the router.push and added redirect in routes -   { path: "/logout", redirect: "/login" }. Now it works fine